src/h5web/visualizations/line/DataCurve.tsx

```diff
@@ -1,22 +1,38 @@
-import React, { Suspense, useMemo } from 'react';
-import { BufferGeometry } from 'three';
+import React, { Suspense, useMemo, useContext } from 'react';
+import { BufferGeometry, Vector3 } from 'three';
 import { Line } from 'react-three-fiber/components';
-import { useDataPoints } from './hooks';
+import { useThree } from 'react-three-fiber';
 import { useLineConfig } from './config';
 import { CurveType } from './models';
 import DataGlyphs from './DataGlyphs';
+import { AxisSystemContext } from '../shared/AxisSystemProvider';
+import { getAxisScale } from '../shared/utils';
 
 const COLOR = '#1b998b';
 
-function DataCurve(): JSX.Element {
-  const points = useDataPoints();
+interface Props {
+  values: number[];
+}
```

Comment:
Just a minor thing: now that `DataCurve` accepts props I would put the `color` in it as an optional prop (with a fallback to `#12998b`)